api/nodes/views.py

```diff
@@ -665,15 +666,15 @@ class NodeChildrenList(JSONAPIBaseView, bulk_views.ListBulkCreateJSONAPIView, No
     ordering = ('-modified',)
 
     def get_default_queryset(self):
-        return default_node_list_queryset(model_cls=Node)
+        auth = get_user_auth(self.request)
+        return default_node_list_permission_queryset(auth=auth, model_cls=Node)
 
     # overrides ListBulkCreateJSONAPIView
     def get_queryset(self):
         node = self.get_node()
-        auth = get_user_auth(self.request)
         node_pks = node.node_relations.filter(is_node_link=False).select_related('child')\
                 .values_list('child__pk', flat=True)
-        return self.get_queryset_from_request().filter(pk__in=node_pks).can_view(auth.user).order_by('-modified')
+        return self.get_queryset_from_request().filter(pk__in=node_pks).order_by('-modified')
```

Comment:
`get_queryset_from_request` calls `get_default_queryset` at some point, which already ensures that the user has view permissions to the nodes returned.  Those nodes are further filtered on nodes that are children.  There is no need to call `can_view` again.